libcontainer/cgroups/fs/stats_util_test.go

```diff
@@ -26,98 +25,109 @@ func blkioStatEntryEquals(expected, actual []cgroups.BlkioStatEntry) error {
 
 func expectBlkioStatsEquals(t *testing.T, expected, actual cgroups.BlkioStats) {
 	if err := blkioStatEntryEquals(expected.IoServiceBytesRecursive, actual.IoServiceBytesRecursive); err != nil {
-		logrus.Printf("blkio IoServiceBytesRecursive do not match - %s\n", err)
-		t.Fail()
+		t.Errorf("blkio IoServiceBytesRecursive do not match - %s\n", err)
```

Comment:
Please move this cleanup out to a separate commit that goes first.

Reply:
Aren't all PR's commits squashed before merge anyway?